bitnami/postgresql/templates/_helpers.tpl

```diff
@@ -310,21 +310,33 @@ postgresql: tls.enabled, volumePermissions.enabled
 Return the path to the cert file.
 */}}
 {{- define "postgresql.tlsCert" -}}
-{{- required "Certificate filename is required when TLS in enabled" .Values.tls.certFilename | printf "/opt/bitnami/postgresql/certs/%s" -}}
+{{- if and .Values.tls.enabled .Values.tls.autoGenerated }}
```

Comment:
Is it necessary to check that `.Values.tls.enabled` is `true`? I thought it was implicit, I mean this macro is only invoked under `{{- if .Values.tls.enabled }}` blocks.